### ISSUE TITLE
Species lacking regional distribution

### ISSUE THREAD

**Issue by SanderDevisscher** [trias-project/unified-checklist#45]
Some species are currently lacking the flemish region locality on the "include-distribution-regions" - branch. These include:

| Species              | Key     |
|----------------------|---------|
| Alopochen aegyptiaca | 2498252 |
| Oxyura jamaicensis   | 2498305 |
| Ludwigia grandiflora | 5421039 |
| Ludwigia peploides   | 5420991 |

According to Tim the missing birds can be explained by the fact the birds checklist is not yet published. This also explains the missing of Sacred ibis from the unified checklist.

**Comment by damianooldoni** [comment]
Yes, based on the data from the checklists we use to build the unified checklist there is no distribution with locality equal to Flanders. 
For example, Alopochen aegyptiaca is only present in checklist _RINSE - Pathways and vectors of biological invasions in Northwest Europe_: see [this link](https://www.gbif.org/species/search?q=Alopochen%20aegyptiaca&dataset_key=0a2eaf0c-5504-4f48-a47f-c94229029dc8&dataset_key=1f3505cd-5d98-4e23-bd3b-ffe59d05d7c2&dataset_key=1738f272-6b5d-4f43-9a92-453a8c5ea50a&dataset_key=9ff7d317-609b-4c08-bd86-3bc404b77c42&dataset_key=3c428404-893c-44da-bb4a-6c19d8fb676a&dataset_key=98940a79-2bf1-46e6-afd6-ba2e85a26f9f&dataset_key=b043c480-dd36-4f4f-aa82-e188753ff09d&dataset_key=e082b10e-476f-43c1-aa61-f8d92f33029a&advanced=1).
This taxon has the following distributions:
http://api.gbif.org/v1/species/146042280/distributions
No Flanders, only Belgium,as you can see.

I agree with @timadriaens: publishing a bird checklist would surely improve the data in the unified checklist.
I remove the bug label, as it is not a bug in (the code producing) the unified checklist.
Notice how the subspecies _Ludwigia peploides subsp. montevidensis_ has a Flemish distribution, as this taxon is present in _The Manual Of The Alien Plants in Belgium_ (see [search link](https://www.gbif.org/species/search?q=Ludwigia%20peploides%20montevidensis&dataset_key=0a2eaf0c-5504-4f48-a47f-c94229029dc8&dataset_key=1f3505cd-5d98-4e23-bd3b-ffe59d05d7c2&dataset_key=1738f272-6b5d-4f43-9a92-453a8c5ea50a&dataset_key=9ff7d317-609b-4c08-bd86-3bc404b77c42&dataset_key=3c428404-893c-44da-bb4a-6c19d8fb676a&dataset_key=98940a79-2bf1-46e6-afd6-ba2e85a26f9f&dataset_key=b043c480-dd36-4f4f-aa82-e188753ff09d&dataset_key=e082b10e-476f-43c1-aa61-f8d92f33029a&advanced=1)). This taxon has a Belgian, Flemish and Walloon distribution: see [distributions via API](http://api.gbif.org/v1/species/141266547/distributions). And all these three distributions are present in [`data/processed/distribution.csv`](https://raw.githubusercontent.com/trias-project/unified-checklist/include-distribution-regions/data/processed/distribution.csv) in branch `include-distribution-regions`. So, happy to see again that changes in this branch seem ok.

**Comment by timadriaens** [comment]
Indeed, the bird checklist will solve this problem. For the plants: _L. grandiflora_ is on MAP as **_Ludwigia hexapetala (Hook. & Arnott) Zardini, Gu & Raven_** with grandiflora as a synonym and does have a distribution for FLA àand WAL. So it should be there. 
--
